### ISSUE TITLE
Static content deploy in 2.1.0 fails as newer flags are not avaiable

### ISSUE THREAD

**Issue by jpratt** [davidalger/capistrano-magento2#45]
When running the deployment it fails on version 2.1.0 as specific content flags such as --no-javascript or --no-css are not valid until v2.1.1. Same issue if you use the :magento_deploy_themes option as it is also not available until 2.1.1. Both documentation and code should reflect this version. Created pull request: https://github.com/davidalger/capistrano-magento2/pull/46

**Comment by davidalger** [comment]
@jpratt Thanks for the feedback. I don't think my team here at CL every got to deploying 2.1.0 to production before 2.1.1 dropped, but my mistake entirely!

I merged your PR into develop and then updated the version checks to explicitly check the patch release version so the change won't halt the use of these flags on 2.1.1 and later.

The issue with :magento_deploy_themes causing a failure on 2.1.0 vs a warning has also been fixed.

See the above tagged commit for details. The CL and README have been updated in develop. A release which contains these updates will be up soon.
